cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -46,10 +45,10 @@ import (
 	"golang.org/x/exp/maps"
 )
 
-var Defaults = Options{
-	Module: "main",
-	Path:   "infra",
-}
+const (
```

Comment:
Could we not just export the ones in the provisioning package and consume them here?  It feels a little fragile to have the same value as a constant but duplicated across so many packages.

Reply:
The main intention is to break the dependency from one package to another, when it comes to default values.
Each package should know what are the default values for its own types.